extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -629,6 +629,7 @@ export const targetPlatformToVersion: Map<string, string> = new Map<string, stri
 	[SqlTargetPlatform.sqlServer2022, '160'],
 	[SqlTargetPlatform.sqlAzure, 'AzureV12'],
 	[SqlTargetPlatform.sqlDW, 'Dw'],
+	[SqlTargetPlatform.sqlDwServerless, 'Serverless'],
```

Comment:
This maps to Database Schema Provider in the .sqlproj, correct?  If so, can you update the test that validates support? https://msdata.visualstudio.com/SQLToolsAndLibraries/_git/SqlProjects?path=/Test/ProjectTests.cs&version=GBmain&line=201&lineEnd=201&lineStartColumn=27&lineEndColumn=52&lineStyle=plain&_a=contents

Reply:
Added https://msdata.visualstudio.com/SQLToolsAndLibraries/_git/SqlProjects/pullrequest/1147824